app/admin/RTMediaAdmin.php

```diff
@@ -178,35 +178,9 @@ public function __construct() {
 		public function install_transcoder_admin_notice() {
 			$show_notice = get_site_option( 'install_transcoder_admin_notice', 1 );
 			if ( '1' === $show_notice || 1 === $show_notice ) :
-				?>
-				<div class="notice notice-info install-transcoder-notice is-dismissible">
-					<?php wp_nonce_field( '_install_transcoder_hide_notice_', 'install_transcoder_hide_notice_nonce' ); ?>
-					<p>
-						<?php
-						$allowed_tags = array(
-							'a' => array(
-								'href'   => array(),
-								'target' => array(),
-							),
-						);
-						echo wp_kses( __( 'Install <a href="https://wordpress.org/plugins/transcoder/" target="_blank">Transcoder plugin</a> to convert audio/video files and thumbnails generation.', 'buddypress-media' ), $allowed_tags );
-						?>
-					</p>
-				</div>
-				<script type="text/javascript">
-					jQuery( document ).ready( function() {
-						jQuery( '.install-transcoder-notice.is-dismissible' ).on( 'click', '.notice-dismiss', function() {
-							var data = {
-								action: 'install_transcoder_hide_admin_notice',
-								install_transcoder_notice_nonce: jQuery('#install_transcoder_hide_notice_nonce').val()
-							};
-							jQuery.post( ajaxurl, data, function ( response ) {
-								jQuery('.install-transcoder-notice').remove();
-							});
-						});
-					});
-				</script>
-				<?php
+
+				include RTMEDIA_PATH . 'app/admin/templates/notices/transcoder.php';
```

Comment:
Use `require_once` because this file need to load only once so. Address it for other new templates.